controllers/binding/snapshotenvironmentbinding_adapter.go

```diff
@@ -162,9 +163,11 @@ func (a *Adapter) createIntegrationPipelineRunWithEnvironment(application *appli
 		WithIntegrationLabels(integrationTestScenario).
 		WithEnvironmentAndDeploymentTarget(deploymentTarget, environment.Name).
 		AsPipelineRun()
+
 	// copy PipelineRun PAC annotations/labels from snapshot to integration test PipelineRuns
-	h.CopyAnnotationsByPrefix(&snapshot.ObjectMeta, &pipelineRun.ObjectMeta, gitops.PipelinesAsCodePrefix, gitops.PipelinesAsCodePrefix)
-	h.CopyLabelsByPrefix(&snapshot.ObjectMeta, &pipelineRun.ObjectMeta, gitops.PipelinesAsCodePrefix, gitops.PipelinesAsCodePrefix)
+	_ = metadata.CopyAnnotationsByPrefix(&snapshot.ObjectMeta, &pipelineRun.ObjectMeta, gitops.PipelinesAsCodePrefix)
```

Comment:
I see that previous function version used 4 inputs and new one uses 3, but I don't se updated function definition, is it planned in idfferent PR?

Reply:
Currently, we are using functions from [here](https://github.com/redhat-appstudio/operator-toolkit/blob/main/metadata/metadata.go).